libs/design-system/icon-button/src/lib/icon-button.stories.ts

```diff
@@ -1,10 +1,10 @@
-import { provideHttpClient } from '@angular/common/http';
 import { MatButtonModule } from '@angular/material/button';
 import { MatIconModule } from '@angular/material/icon';
-import { provideIcons } from '@hra-ui/cdk/icons';
-import { applicationConfig, moduleMetadata, type Meta, type StoryObj } from '@storybook/angular';
+import { provideDesignSystem } from '@hra-ui/design-system';
+import { applicationConfig, Meta, moduleMetadata, StoryObj } from '@storybook/angular';
+
 import { IconButtonSizeDirective } from './icon-button-size/icon-button-size.directive';
-import { provideIconButtons } from './providers';
+import { IconButtonVariantDirective } from './icon-button-variant/icon-button-variant.directive';
 
 const meta: Meta = {
   title: 'IconButton',
```

Comment:
Update argtypes with a variant input (like the size input but different options)